pkg/topology/topology.go

```diff
@@ -52,7 +52,7 @@ type Hint struct {
 type Hints map[string]Hint
 
 func getDevicesFromVirtual(realDevPath string) (devs []string, err error) {
-	if !filepath.HasPrefix(realDevPath, "/sys/devices/virtual") {
+	if !strings.HasPrefix(realDevPath, "/sys/devices/virtual") {
```

Comment:
IIRC our preference has been `filepath`. What's the linter error?

Reply:
The error was about `filepath.HasPrefix()` being deprecated.

Reply:
indeed! maybe `filepath.Rel()`?

Reply:
Yeap. I couldn't get rid of `strings.HasPrefix()`, but with `filepath.Rel()` not ignoring errors it seems to be safer now.